### ISSUE TITLE
Publish to a repo

### ISSUE THREAD

**Issue by andreaTP** [reactive-streams/reactive-streams-scalajs#2]
Please could you publish "version-1" branch at least on a snapshots repo?

**Comment by alexandru** [comment]
I'd like that, but conversation got stuck in PR #1 :-(

What project are you working on that needs it btw? :-)

**Comment by andreaTP** [comment]
haven't read that 'till the end, could you summarize into a proper issue what is the blocking point so that someone at somepoint could pick it up? ;-)

Thanks a lot!

btw:
https://github.com/unicredit/akka.js/tree/stream

**Comment by alexandru** [comment]
I'm guessing lack of time or interest, summer time & all. I recommend 
commenting on that PR.

On 2016-07-05 15:33, Andrea Peruffo wrote:

> haven't read that 'till the end, could you summarize into a proper 
> issue what is the blocking point so that someone at somepoint could 
> pick it up? ;-)
> 
> Thanks a lot!
> 
> btw:
> https://github.com/unicredit/akka.js/tree/stream
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> https://github.com/reactive-streams/reactive-streams-scalajs/issues/2#issuecomment-230465289, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/AAAt6a26Gi-LDSwZ7nAUvZ_Rn2QHy_dmks5qSk8bgaJpZM4JFD1N.

## 

Alexandru Nedelcu
alexn.org https://alexn.org
